### DISCUSSION TITLE
Welcome to obsidian-theme-ebullientworks Discussions!

### DISCUSSION THREAD

**Discussion by ebullient** [ebullient/obsidian-theme-ebullientworks#7] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
* 🙋 [Ask questions](/discussions/categories/q-a).
* 💡 [Share ideas](/discussions/categories/ideas).
* 💬 Engage with other community members in [General](/discussions/categories/general)
* 🎉 [Show your stuff!](/discussions/categories/show-and-tell)
* Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by KzBeck** [comment]
The tags for me (using a light version) are too dark to read. I've tried to find where I can change them, but as of today I haven't found it.  Currently, it is a reddish color on black.  I need more contrast. The hover is black on black. 

I appreciate the help!

Voted Ebullientworks as a GEM for 2021!

**Reply by ebullient** [reply]
Can you snip a picture? 

Tags in light mode (selected): 
<img width="108" alt="image" src="https://user-images.githubusercontent.com/808713/148070642-5fa64502-821f-48bd-8921-9b73a745dc92.png">

Tags in dark mode are dark, yes: 
<img width="133" alt="image" src="https://user-images.githubusercontent.com/808713/148070724-b554658f-f8b8-4615-a849-b02071de09e7.png">

Did you mean dark mode?

**Reply by ebullient** [reply]
Converted this to an issue and pushed a toggle to allow you to fiddle. ;) Readme updated as well.

**Comment by KzBeck** [comment]
No, light mode here are a couple of pics

![2022-01-04_12-06-16](https://user-images.githubusercontent.com/51794712/148096774-cfba16f1-71ca-4857-8774-c706fc78471a.png)

![2022-01-04_12-05-49](https://user-images.githubusercontent.com/51794712/148096785-4be4e9d0-b338-4395-8d79-21431dffd04d.png)

**Reply by ebullient** [reply]
AH! That's a style collision. I don't have tag pills in my theme, so I think you have another snippet active that is creating the black background. Do you have any snippets enabled?

Have you updated the theme lately? You might go to the community themes settings in Obsidian and update it to make sure you have the latest.

Let's move additional conversation on this to #10 .. so we can hash it out and I can fix more things if necessary (though I believe you either have an old! version, or a snippet clash). =)
